### ISSUE TITLE
Accessibility - iOS - ComplyChain Application Logo has Color Contrast Problem

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#90]
The blue/purplish color of the words "**ILAB**" for the iPad icon only have a color contrast ratio of 2.88:1 and needs to be 4.5:1 or higher to pass. 

Also, it's so jammed into the inside left part of the gear it's hard to read that it says  "**ILAB**". Please make the text white and moving the "B" to the outside edge of where the right-side of the gear would make it easier to read. Not sure why the iOS ComplyChain icon is on what looks to be a black background.

See example on our iPad device with the default device/operating system background.

![complychain - logo - color contrast issue](https://user-images.githubusercontent.com/10423487/42176197-2073d7f4-7df6-11e8-9ced-d42ac1b20900.png)

**Comment by sanganalakshay** [comment]
fixed

**Comment by jfc3-dol** [comment]
The newly updated ComplyChain application icon has enough color contrast and is now easier to see and read. Thanks, for adding the white background and moving the text a bit to the right.

See the screenshot below.

![complychain device icon](https://user-images.githubusercontent.com/10423487/43418346-9c430e10-940b-11e8-85d1-ae75e68235bd.png)
